src/collections/_documentation/development/sdk-dev/unified-api.md

```diff
@@ -156,16 +173,16 @@ The SDK maintains two variables: The *main hub* (a global variable) and the *cur
 
 A scope holds data that should implicitly be sent with Sentry events. It can hold context data, extra parameters, level overrides, fingerprints etc.
 
-The user should be able to modify the current scope easily (to set extra, tags, current user), through a global function `configure_scope`.  `configure_scope` takes a callback function to which it passes the current scope. Here's an example from another place in the docs:
+The user can modify the current scope (to set extra, tags, current user) through the global function `configure_scope`.  `configure_scope` takes a callback function to which it passes the current scope.
+If the SDK is disabled, Sentry may not invoke the callback.
+
+Here's an example from another place in the docs:
 
 ```javascript
-Sentry.configureScope(function(scope) {
-  scope.setExtra("character_name", "Mighty Fighter");
-});
+Sentry.configureScope(scope =>
+  scope.setExtra("character_name", "Mighty Fighter"));
 ```
 
-Why not just have a `get_current_scope()` function instead of this indirection?  If the SDK is disabled (e.g. by not providing a DSN), modifying a scope is pointless because there will never be any events to send. In that situation `configure_scope` may choose not to call the callback.  This is also used to more efficiently flush out scope changes in some languages.  When a `with` statement is used that always executes the scope needs to be a dummy scope object.  It also helps to emphasize that under normal circumstances we do not want the user the do the scope handling.
```

Comment:
Why remove this completely? In general it is good to have insights on why certain APIs even exist.